pom.xml

```diff
@@ -16,7 +16,7 @@
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <maven.compiler.source>11</maven.compiler.source>
         <maven.compiler.target>11</maven.compiler.target>
-        <kafka.version>2.8.1</kafka.version>
+        <kafka.version>3.4.0</kafka.version>
```

Comment:
Dependency major version raise, will it impact minor expectations?

Reply:
Valid concern, I will revert this change. Doesn't make sense to update the major version yet.